Algorithm.CSharp/WeeklyUniverseSelectionRegressionAlgorithm.cs

```diff
@@ -26,22 +26,22 @@ namespace QuantConnect.Algorithm.CSharp
     /// <meta name="tag" content="regression test" />
     public class WeeklyUniverseSelectionRegressionAlgorithm : QCAlgorithm
     {
-        private readonly Symbol _ibm = QuantConnect.Symbol.Create("IBM", SecurityType.Equity, Market.USA);
         private SecurityChanges _changes = SecurityChanges.None;
 
         /// <summary>
         /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
         /// </summary>
         public override void Initialize()
         {
-            UniverseSettings.Resolution = Resolution.Hour;
-
             SetStartDate(2013, 10, 1);  //Set Start Date
             SetEndDate(2013, 10, 31);    //Set End Date
             SetCash(100000);             //Set Strategy Cash
 
+            SetBenchmark(dt => 1m);
```

Comment:
Ideally we could have this running without this `SetBenchmark` line -- I do realize that that with the 2% change the benchmark dependent stats will change, but the orders/fills/security changes should be identical.

Reply:
Ok, removed SetBenchmark call